src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml

```diff
@@ -542,10 +542,8 @@
                                             <RowDefinition Height="Auto" />
                                         </Grid.RowDefinitions>
                                         <Button x:Name="AddStyleButton"
-                                                Style="{StaticResource FlatButtonStyle}" 
+                                                Style="{StaticResource SolidButtonStyleSmaller}" 
```

Comment:
Do we also need to apply this style for `ResetCPython` button?

Reply:
@QilongTang the `ResetCPython `button is using the style `SolidButtonStyle`, the style that I added SolidButtonStyleSmaller is almost the same that `SolidButtonStyle `the only changes are in a Border.Height, TextBlock.FontSize and the Background color triggered when the Button is disabled (based in the Figma design provided by Jingyi). I guess the FontSize/Height is smaller due that the AddStyle button is in a smaller space than the other buttons.
Here you can see a GIF of the two buttons
![buttons_style](https://user-images.githubusercontent.com/61755417/160145738-2f033867-dff8-492d-9b1b-26fb6f6d36b2.gif)

Reply:
@QilongTang comment added in the next commit:
https://github.com/DynamoDS/Dynamo/pull/12738/commits/180c9b1dade33077256f9278733cf40491bd6e7f